copilot/fsd-frontend/manifest.yml

```diff
@@ -45,39 +45,55 @@ network:
 #
 # Pass environment variables as key value pairs.
 variables:
-  SENTRY_DSN: "https://3c6d2fd1e3824aecb3826a7d640b29a9@o1432034.ingest.sentry.io/4503897449103360"
-  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
-  COOKIE_DOMAIN: ".test.levellingup.gov.uk"
-  APPLICANT_FRONTEND_HOST: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
-  AUTHENTICATOR_HOST: "https://fsd:fsd@authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" # pragma: allowlist secret
   ACCOUNT_STORE_API_HOST: "http://fsd-account-store:8080"
+  APPLICANT_FRONTEND_HOST: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
   APPLICATION_STORE_API_HOST: "http://fsd-application-store:8080"
-
-  FORMS_SERVICE_PUBLIC_HOST: "https://fsd:fsd@forms.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" #pragma: allowlist secret
+  AUTHENTICATOR_HOST: "https://fsd:fsd@authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" # pragma: allowlist secret
+  COOKIE_DOMAIN: ".test.levellingup.gov.uk"
+  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
   FORMS_SERVICE_PRIVATE_HOST: "https://fsd:fsd@forms.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" #pragma: allowlist secret
-
+  FORMS_SERVICE_PUBLIC_HOST: "https://fsd:fsd@forms.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk" #pragma: allowlist secret
   FUND_STORE_API_HOST: "http://fsd-fund-store:8080"
   NOTIFICATION_SERVICE_HOST: http://fsd-notification:8080
   REDIS_INSTANCE_URI:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-MagicLinksRedisInstanceURI
+  SENTRY_DSN: "https://3c6d2fd1e3824aecb3826a7d640b29a9@o1432034.ingest.sentry.io/4503897449103360"
 
 secrets:
   RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PUBLIC_KEY_BASE64
 
 # You can override any of the values defined above by environment.
 environments:
-  # uat:
-    # http:
-    #   alias: frontend.uat.access-funding.test.levellingup.gov.uk
```

Comment:
Are we waiting for DNS before uncommenting this? Do we want to add a TODO.

Reply:
This is being removed, the default alias takes care of this

Reply:
OK, makes sense, I guess that means though we'll need to line up IT to point to: `${SvcName}.${EnvName}.${AppName}.${DomainName}`

Reply:
That's correct - that's already in the prerequisites document, although all except production are done now.